src/v2/api/resources/ResourcesService.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { QueryAllDTO } from 'src/v2/utils/QueryAllDTO';
+import { CreateResourceDTO } from './dto/CreateResourceDTO';
+import { UpdateResourceDTO } from './dto/UpdateResourceDTO';
+import { ResourceRepository } from './ResourcesRepository';
+
+@Injectable()
+export class ResourceService {
+  constructor(
+    private resourceRepository: ResourceRepository
+  ) {}
+
+  async getAll(body: QueryAllDTO) {
+    const studentResources = await this.resourceRepository.getAll(body); 
+    return { studentResources };
```

Comment:
you make an `studentResources` wrapper both in a controller and service. you need to do it only in controller